tools/build_api.py

```diff
@@ -331,14 +331,6 @@ def is_official_target(target_name, version):
                     ("Currently it is only configured to support the ") + \
                     ("following toolchains: %s" %
                      ", ".join(sorted(supported_toolchains)))
-
-            elif not target.default_lib == 'std':
-                result = False
-                reason = ("Target '%s' must set the " % target.name) + \
```

Comment:
Where do we now show this warning?

Reply:
This warning is not required any more as we implied this value in toolchains.